Makefile

```diff
@@ -1,81 +1,65 @@
 docker-tests: 
 	cd test; python3 -m venv python_env
-	cd test; source ./python_env/bin/activate; pip3 install -r requirements.txt; robot -x docker-tests.xml --outputdir test_results --variable TEST_IMAGE:${test_image} --variable MARKLOGIC_VERSION:${version} --variable BUILD_BRANCH:${build_branch} --maxerrorlines 9999 ./docker-tests.robot; deactivate
+	cd test; source ./python_env/bin/activate; pip3 install -r requirements.txt; robot -x docker-tests.xml --outputdir test_results --variable TEST_IMAGE:${current_image} --variable MARKLOGIC_VERSION:${version} --variable BUILD_BRANCH:${build_branch} --maxerrorlines 9999 ./docker-tests.robot; deactivate
 	rm -r test/python_env/
 	
 #***************************************************************************
 # run all tests
 #***************************************************************************
 .PHONY: test
 test: structure-test docker-tests
-	
-#***************************************************************************
-# push docker images to mlregistry.marklogic.com
-#***************************************************************************
-push-mlregistry:
-	docker tag ${REPONAME}/marklogic-server-centos:${version} ${docker_registry}/${repoDir}/marklogic-server-centos:${version}
-	docker push ${docker_registry}/${repoDir}/marklogic-server-centos:${version}
 
 #***************************************************************************
-# run lint checker on Dockerfiles, print linting issues but do not fail the build
+# run lint checker on shell scripts and Dockerfiles, print linting issues but do not fail the build
 #***************************************************************************
+
 lint:
-	docker run --rm -v "${PWD}:/mnt" koalaman/shellcheck:stable src/scripts/start-marklogic.sh $(if $(Jenkins), > start-marklogic-lint.txt,)
-	docker run --rm -v "${PWD}:/mnt" koalaman/shellcheck:stable src/scripts/start-marklogic-rootless.sh $(if $(Jenkins), >> start-marklogic-lint.txt,)
-	docker run --rm -i -v "${PWD}/hadolint.yaml":/.config/hadolint.yaml ghcr.io/hadolint/hadolint < dockerFiles/marklogic-deps-centos:base $(if $(Jenkins), > marklogic-deps-centos-base-lint.txt,)
-	docker run --rm -i -v "${PWD}/hadolint.yaml":/.config/hadolint.yaml ghcr.io/hadolint/hadolint < dockerFiles/marklogic-server-centos:base $(if $(Jenkins), > marklogic-server-centos-base-lint.txt,)
-	docker run --rm -i -v "${PWD}/hadolint.yaml":/.config/hadolint.yaml ghcr.io/hadolint/hadolint < dockerFiles/marklogic-server-ubi:base $(if $(Jenkins), >> marklogic-server-centos-base-lint.txt,)
-	docker run --rm -i -v "${PWD}/hadolint.yaml":/.config/hadolint.yaml ghcr.io/hadolint/hadolint < dockerFiles/marklogic-server-ubi-rootless:base $(if $(Jenkins), >> marklogic-server-centos-base-lint.txt,)
+	docker run --rm -v "${PWD}:/mnt" koalaman/shellcheck:stable src/scripts/*.sh $(if $(Jenkins), > start-scripts-lint.txt,)
+
+	@for dockerFile in $(shell ls ./dockerFiles/); do\
```

Comment:
Maybe we shouldn't scan all images on very run?

Reply:
The recommended process is to scan image at every build.
But we can stick to centOS and UBI it is not needed to scan the rootless as based on UBI.

Reply:
Yeah, currently this will scan every single docker file in the dockerFile folder on every build. It adds a bit of noise to the build notification email though.